management/server/policy.go

```diff
@@ -0,0 +1,421 @@
+package server
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"fmt"
+	"html/template"
+	"strings"
+
+	"github.com/netbirdio/netbird/management/server/activity"
+	"github.com/netbirdio/netbird/management/server/status"
+
+	"github.com/open-policy-agent/opa/rego"
+	log "github.com/sirupsen/logrus"
+)
+
+// PolicyUpdateOperationType operation type
+type PolicyUpdateOperationType int
+
+// PolicyTrafficActionType action type for the firewall
+type PolicyTrafficActionType string
+
+const (
+	// PolicyTrafficActionAccept indicates that the traffic is accepted
+	PolicyTrafficActionAccept = PolicyTrafficActionType("accept")
+	// PolicyTrafficActionDrop indicates that the traffic is dropped
+	PolicyTrafficActionDrop = PolicyTrafficActionType("drop")
+)
+
+// PolicyUpdateOperation operation object with type and values to be applied
+type PolicyUpdateOperation struct {
+	Type   PolicyUpdateOperationType
+	Values []string
+}
+
+//go:embed rego/default_policy_module.rego
+var defaultPolicyModule string
+
+//go:embed rego/default_policy.rego
+var defaultPolicyText string
+
+// defaultPolicyTemplate is a template for the default policy
+var defaultPolicyTemplate = template.Must(template.New("policy").Parse(defaultPolicyText))
+
+// PolicyRule is the metadata of the policy
+type PolicyRule struct {
+	// ID of the policy rule
+	ID string
+
+	// Name of the rule visible in the UI
+	Name string
+
+	// Description of the rule visible in the UI
+	Description string
+
+	// Enabled status of rule in the system
+	Enabled bool
+
+	// Action policy accept or drops packets
+	Action PolicyTrafficActionType
+
+	// Destinations policy destination groups
+	Destinations []string
+
+	// Sources policy source groups
+	Sources []string
+}
+
+// Copy returns a copy of a policy rule
+func (pm *PolicyRule) Copy() *PolicyRule {
+	return &PolicyRule{
+		ID:           pm.ID,
+		Name:         pm.Name,
+		Description:  pm.Description,
+		Enabled:      pm.Enabled,
+		Action:       pm.Action,
+		Destinations: pm.Destinations[:],
+		Sources:      pm.Sources[:],
+	}
+}
+
+// ToRule converts the policy rule to a rule.
```

Comment:
I think that this should be more clear:
`ToRule converts the PolicyRule to a legacy representation of the Rule (for backwards compatibility)`

Reply:
Fixed